README.md

```diff
@@ -35,6 +35,7 @@ Method    | Path
 **GET**   | `/company/{companyNumber}/registered-email-address`                                  | Calls service to retrieve the registered email address for a company
 **GET**   | `/overseas-entity/{companyNumber}/managing-officers`                                 | Calls service to retrieve the managing officers for an overseas entity
 **GET**   | `/overseas-entity/{companyNumber}/beneficial-owners`                                 | Calls service to retrieve the beneficial owner data for an overseas entity
+**GET**   | `/overseas-entity/trusts/{trustId}/individual-trustees`                              | Calls service to retrieve the individual trustee data for a trust
```

Comment:
I'm only passing in trust id as that is all that is required for the query- should i change this to use company number as well?

Reply:
I suppose it would be more for debugging purposes to know which OE this query refers to and we'd be following the same pattern of the existing endpoints where the company number is always passed in

Reply:
I don't feel it's needed. If we do we'd have to check that the trust is actually under the OE?

Reply:
We may want to think about the endpoint we create on the ROE API and api sdk too as I think we were planning on including the OE number, but as you say if we're not checking the trust is under the OE it may not be needed. 

Happy to leave the endpoint as `/overseas-entity/trusts/{trustId}/individual-trustees` for simplicity

Reply:
we will not be adding the oe number in the API but the overseas_entity_id which is the overseas entity submission id in MongoDB - from which we get the OE number.